engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/module/scope/definition/Method.scala

```diff
@@ -49,13 +49,34 @@ object Method {
     */
   sealed case class Explicit(
     override val methodReference: Name.MethodReference,
-    override val body: Expression,
+    val bodyList: Seq[Expression],
+    val isStatic: Boolean,
+    val isStaticWrapperForInstanceMethod: Boolean,
     override val location: Option[IdentifiedLocation],
-    override val passData: MetadataStorage      = MetadataStorage(),
-    override val diagnostics: DiagnosticStorage = DiagnosticStorage()
+    override val passData: MetadataStorage,
+    override val diagnostics: DiagnosticStorage
   ) extends Method
       with IRKind.Primitive {
+    def this(
+      methodReference: Name.MethodReference,
+      body: Expression,
+      location: Option[IdentifiedLocation],
+      passData: MetadataStorage      = MetadataStorage(),
+      diagnostics: DiagnosticStorage = DiagnosticStorage()
+    ) = {
+      this(
+        methodReference,
+        Seq(body),
+        Explicit.computeIsStatic(body),
+        Explicit.computeIsStaticWrapperForInstanceMethod(body),
+        location,
+        passData,
+        diagnostics
+      );
+    }
+
     var id: UUID @Identifier = randomId
+    lazy val body            = bodyList.head
```

Comment:
Same minor nits as with `Lambda`

Reply:
Renamed to `bodySeq` in [13ae27e](https://github.com/enso-org/enso/pull/8207/commits/13ae27e8a86d4b645b5b0be3ecc220f068de1322)